### PR TITLE
Create database

### PR DESCRIPTION
**Pull request by jackiestchen**
Check it out.
These should be enough to get most of the data we need from the db.
If you need to change/add specific query, comment below.

### CODE REVIEW

**Comment by jackiestchen** [comment]
Need to remove test codes before merge

File: db/schema/01_schema.sql [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  isOwner BOOLEAN DEFAULT FALSE
+);
+
+CREATE TABLE items (
+  id SERIAL PRIMARY KEY NOT NULL,
+  name VARCHAR(255) NOT NULL,
+  description TEXT,
+  price INT NOT NULL,
+  image_url TEXT
+);
```
Comment (isissi) [inline db/schema/01_schema.sql:22]: I'm thinking maybe can add one "category" column in items table so the items in the menu can get categorized? Could be like "Bubble Tea", "Fruit Tea", "Tea Latte", "Slush", etc.

File: db/seeds/02_items.sql [hunk 2]
```diff
@@ -0,0 +1,43 @@
+INSERT INTO items (
+  name, description, price, image_url
+) VALUES (
+  'Classic', 'Signature bubble tea with 3 toppings: bubbles, pudding and grass jelly', 6.99, 'https://images.squarespace-cdn.com/content/v1/5edfb8e7559cd9129deae815/1629474098758-PX9OU33P0MIPXHMDD4JL/2+Guys.png'
```
Comment (isissi) [inline db/seeds/02_items.sql]: I'll look for the product images and create an images folder in public. The path will be "/images/image_name". Not sure if it's good to use pics with other logo on it lol

File: db/seeds/02_items.sql [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+INSERT INTO items (
+  name, description, price, image_url
+) VALUES (
+  'Mango Slush',  'Fresh milk blended with ice and mango for a perfect balance of slushy and chewy bobab', 6.99, 'https://images.squarespace-cdn.com/content/v1/5edfb8e7559cd9129deae815/1607618450914-QVSBI4WFA7BQHHVJORD6/Mango+Slush'
+);
+INSERT INTO items (
+  name, description, price, image_url
+) VALUES (
+  'Black Sugar Milk w/ Cream Mouse',  'This drink is formulated with precision. Our sugar and ice level are perfectly balanced to deliver the most delicious drink to you.', 7.99, 'https://s3-media0.fl.yelpcdn.com/bphoto/4jZEfsUv0LuWdBXr_RHZuA/l.jpg'
```
Comment (isissi) [inline db/seeds/02_items.sql]: Guess it's a typo lol is it mousse?

File: routes/databaseRoutes.js [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+    .query(queryString, values)
+    .then((res) => {
+      return res.rows[0];
+    })
+    .catch((err) => console.log(err));
+};
+
+exports.addUser = addUser;
+
+const getAllOrders = (user_id, limit = 10) => {
```
Comment (isissi) [inline routes/databaseRoutes.js]: We may need getUserOrders for user, and getOpenOrders & getHistoryOrders for admin?

**Review by isissi — COMMENTED** [review]
I left comments in 02_items.sql & databaseRoutes.js. I've also added an images folder under public with product images & logo so you can add the paths.

File: db/seeds/02_items.sql (hunk 3 above)
Comment (jackiestchen) [reply db/seeds/02_items.sql]: Yup. Typo. lol

File: db/seeds/02_items.sql (hunk 2 above)
Comment (jackiestchen) [reply db/seeds/02_items.sql]: Yup. I just put something in there for sample data.

Comment (jackiestchen) [reply db/seeds/02_items.sql]: Nice pics!

**Pull request merged**